### DISCUSSION TITLE
DateRangePicker 不可选表达式，如何获取moment

### DISCUSSION THREAD

**Discussion by gehaiyi** [alibaba/formily#3008] [Q&A 2.x] [answered]
![image](https://user-images.githubusercontent.com/24401165/161026936-33645c33-01a8-4cb7-9640-0e4d1e21e5bd.png)
DateRangePicker 不可选表达式，如何获取moment，想用moment处理数据

**Comment by janryWang** [comment, marked as answer]
https://github.com/alibaba/formily/discussions/3009
